treescript/tests/test_merges.py

```diff
@@ -178,6 +182,37 @@ def noop_replace(*arguments, **kwargs):
     assert called_args[0] == expected
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "version_config,current_version, expected",
+    (
+        # central-to-beta
+        ({"filename": "config/milestone.txt", "new_suffix": ""}, FirefoxVersion.parse("75.0a1"), "75.0"),
+        ({"filename": "browser/config/version_display.txt", "new_suffix": "b1"}, FirefoxVersion.parse("75.0a1"), "75.0b1"),
```

Comment:
I like the fact that this suite is exhaustive! The number of files is small and this test doesn't seem too expensive to run, so I agree we can be as exhaustive as possible.

Nit: I found an additional case:

```suggestion
        ({"filename": "browser/config/version.txt", "new_suffix": ""}, FirefoxVersion.parse("75.0a1"), "75.0"),
        ({"filename": "browser/config/version_display.txt", "new_suffix": "b1"}, FirefoxVersion.parse("75.0a1"), "75.0b1"),
```

Reply:
I think that's the same replacement operation as `({"filename": "config/milestone.txt", "new_suffix": ""}, FirefoxVersion.parse("75.0a1"), "75.0"),` isn't it? Since the test mocks out reading the actual file, which doesn't exist.

Reply:
True. I was thinking on making this case explicit so the expected behavior is documented by the test. I agree with you, from a functional perspective, it doesn't test anything more than the previous case.